codegen/facelift-codegen.py

```diff
@@ -64,32 +61,30 @@ def getPrimitiveCppType(symbol):
         return 'float'
     return symbol;
 
-def qmlCompatibleType(symbol):
-    if symbol.type.is_interface:
+def qmlCompatibleType(self):
```

Comment:
I don't like the argument name "self". Isn't it conventionally used in Python as the "this pointer". Though we are not inside a class here.

Reply:
We are in a class since that function is added to a class dynamically